### DISCUSSION TITLE
FFAPI stops background music/Jukeboxes from playing?

### DISCUSSION THREAD

**Discussion by Tursiops-G** [Sinytra/ForgifiedFabricAPI#16] [Q&A] [answered]
This is Plain Forge v47.1.3 & FFAPI 0.86.1+1.8.1+1.20.1 (No other mods).

Before I place an official bug report, Is this a known thing, or is there something I'm missing?

**Comment by Su5eD** [comment]
If jukeboxes work fine without FFAPI installed, then yes, it's a bug indeed.

**Reply by Tursiops-G** [reply, marked as answer]
Bug Report submitted.
Please Delete this thread if it is of no further use/no longer applicable.
Thanks! :)
